test/stdlib/distributions.jl

```diff
@@ -15,6 +15,20 @@ include(dir*"/test/test_utils/AllUtils.jl")
         k = 3
         @test logpdf(d1, k) ≈ logpdf(d2, k)
     end
+
+    @turing_testset "distributions functions" begin
```

Comment:
This test seems to be not enough given that the previous bug was not identified. Wonder if there is any ordered logistic implmeneted in R that we can use for test purpose?

Reply:
The other option is to check the empirical moments against the known moments (if there are nice expressions for these for the `OrderedLogistic` distribution)

Reply:
Would using R and rstan library be a good solution? I could use Stan's functionality to generate random numbers.

Reply:
We're already using `CmdStan.jl` somewhere in our tests, so perhaps give that a go?

Reply:
`CmdStan.jl` is only set up for interface test but the actual command line backend is not installed. 

As this is a discrete distribution, maybe we can validate `rand` using `logpdf`? I.e. draw a lot of r.v.s and check if the emperical distribution is same as what's from `logpdf`.